src/controllers/commands/CommandController.cpp

```diff
@@ -538,6 +539,32 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand("/streamlink", [](const QStringList &words,
+                                            ChannelPtr channel) {
+        if (words.size() < 2)
```

Comment:
Should we also print usage when more than one argument is provided?

Reply:
I don't think so, I was thinking about maybe making something like accepting `best` / `worst` as further arguments and then, after a little rework of streamlink functions, make it autoselect best / worst quality, but I am unsure about it.